File: nijesmik/week04/n^2_배열_자르기.js
function solution(n, left, right) {
  const ans = [];

  for (let i = left; i <= right; i++) {
    const col = i % n;
    const row = (i - col) / n;
    ans.push(Math.max(row, col) + 1);
  }

  return ans;
}

File: damdam6/week04/n^2배열 자르기.java
class Solution {
    public int[] solution(int n, long left, long right) {
        
        int length = (int)(right - left + 1);
        int[] answer = new int[length];
  
        

        for(long i = left; i <=right;i++ ){
            int x = (int) (i/n);
            int y = (int) (i%n);
            answer[(int)(i-left)] = Math.max(x, y)+1;
        }
        
        
        return answer;
    }
}